content/source/docs/enterprise/saml/identity-provider-configuration.html.md

```diff
@@ -11,6 +11,7 @@ Single sign-on setup instructions for specific identity providers (IdP).
 ## Identity Providers
 
 * [OneLogin](./identity-provider-configuration-onelogin.html)
+* [Okta](./identity-provider-configuration-okta.html)
 * [ADFS](./identity-provider-configuration-adfs.html)
```

Comment:
I wonder if we should re-order these to go alphabetically?

Reply:
I'd rather go w/ order of prevalence, but idk if anyone has a good guess about that. Alphabetical is fine w/ me.